src/navier_stokes/VCStaggeredStokesOperator.cpp

```diff
@@ -85,9 +85,6 @@ VCStaggeredStokesOperator::apply(SAMRAIVectorReal<NDIM, double>& x, SAMRAIVector
 {
     IBAMR_TIMER_START(t_apply);
 
-    // Allocate scratch data.
-    d_x->allocateVectorData();
-
```

Comment:
It looks like you forgot one.

Reply:
Just to close the loop --- allocation/deallocation is being handled in the base class.